addons/website/views/snippets/snippets.xml

```diff
@@ -1893,11 +1893,11 @@
                 <we-colorpicker data-name="color_2_opt" title="Secondary"
                                 data-customize-website-color="" data-color="o-color-2"
                                 data-use-css-color="true" data-selected-tab="custom-colors"/>
+                <we-colorpicker data-customize-website-color="" data-color="o-color-3"
```

Comment:
What's the point of that change? Is not that color the one impacting the "light" variable?... should not it stay in the "Light" section ?

Reply:
> Despite we know that's not the case, we're often asked "why it's not possible" to customize eCommerce and other modules UI elements. The correlation color-3 -> $light (+ the fact that $light it's used by the UI), it's apparently obvious only to us... even our PS department sometimes struggle to understand how the color system works (eg. task-3412934).

Yes but it's just normal. I don't expect anyone to guess that but I also can't stop *some* people to complain without even testing (seems to be half the case in the task you mentioned: they just did not test 17.0 and complained about 16.0, seeing their screenshot).

> color-3 is now visually highlighted as a main color. Previously, it was only used for backgrounds, but given its role in defining the UI, it's important for users to recognize its significance.

You can argue this now creates the opposite problem: users see this as a main color, set up a dark red and have their shop broken :shrug: (unless they set up the other ones to matching colors etc I guess but, most of the time it's complicated to put a "real" color there...). It's the case wherever this color setting is placed but I feel like in the "light & dark" section, it was easier to justify its impact.

Anyways, I don't mind the change, you decide, tell me :) But I feel this is an overcompensation (hopefully it's not based just on the fact JEST did not test 17.0 :wink:)

Reply:
> hopefully it's not based just on the fact [...]

Not at all 😉

> You can argue this now creates the opposite problem: users see this as a main color, set up a dark red and have their shop broken 🤷 

Potentially yes (like any other color related issue I'd say..)...  but it's also a problem that they can see, traceback according to their changes and solve... while looking for the right way to customize the UI colors without having a deep understanding of the system could become very frustrating (I've seen POs struggling with their own modules and they're used to work on master -> it rings a bell to me).
IMO these correlations would be clear only after task-3919522 (possibly the original proto in the chatter to be honest).

 > Anyways, I don't mind the change, you decide, tell me :)
 
 Thanks, let's do it 👍